Formula/cling.rb

```diff
@@ -16,15 +16,15 @@ class Cling < Formula
   depends_on "cmake" => :build
 
   resource "clang" do
-    url "http://root.cern.ch/git/clang.git",
-        :tag => "cling-patches-r302975",
-        :revision => "1f8b137c7eb06ed8e321649ef7e3f3e7a96f361c"
+    url "https://github.com/vgvassilev/clang.git",
```

Comment:
Why should we trust you here?

Reply:
Oops, misread your GitHub username as **vgvassilev**. I'm so sorry.

Reply:
Oh, OK — no worries!